packages/terra-demographics-banner/src/_LargeDemographicsBannerDisplay.jsx

```diff
@@ -30,21 +31,16 @@ export default (props) => {
   } = props;
 
   const mainClasses = cx([
-    'demographics-banner',
+    customProps.className,
     'large-demographics-banner',
     { deceased: deceasedDate },
-    customProps.className,
   ]);
 
   delete customProps.className;
 
   return (
     <section className={mainClasses} {...customProps}>
-      {props.photo && (
-        <div className={cx('profile-photo')}>
-          {props.photo}
-        </div>
-      )}
+      <Avatar className={cx('profile-photo')} image={photo} alt={personName} />
```

Comment:
is using personName directly sufficient alt test? Does this need a different description to make sense to screen readers or if the image fails to load?

Reply:
I feel since default avatar would be displayed if the image fails to load, personName would be sufficient for screen readers. would like to know your thoughts on this @neilpfeiffer

Reply:
@emilyrohrbough and @supreethmr, the avatar within the demographics banner should have an empty (alt="") alt attribute if the image is displayed with the user/location name. If the avatar is used as the sole indicator of something, that would be where we would want an actual value within the alt attribute (alt="Bob Jones").